--- a/vhba-module-20240202/vhba.c	2024-02-02 17:43:46.000000000 +0700
+++ b/vhba-module-20240202/vhba.c	2024-08-07 19:45:44.856696601 +0700
@@ -1049,7 +1049,7 @@ static int vhba_probe (struct platform_d
     return 0;
 }
 
-static int vhba_remove (struct platform_device *pdev)
+static void vhba_remove (struct platform_device *pdev)
 {
     struct vhba_host *vhost;
     struct Scsi_Host *shost;
@@ -1061,8 +1061,6 @@ static int vhba_remove (struct platform_
     scsi_host_put(shost);
 
     kfree(vhost->commands);
-
-    return 0;
 }
 
 static void vhba_release (struct device * dev)
